gradle-plugins/compose/build.gradle.kts

```diff
@@ -25,6 +25,7 @@ buildConfig {
     packageName = "org.jetbrains.compose"
     clsName = "ComposeBuildConfig"
     buildConfigField("String", "composeVersion", BuildProperties.composeVersion(project))
+    buildConfigField("Boolean", "isComposeWithWeb", BuildProperties.isComposeWithWeb(project).toString())
```

Comment:
Why this property make sense? Shalln't we make it marked for removal.

Reply:
This is simply a build time constant explicitly telling to build `with-web`. We can check the version of course, but I would prefer not to do that, because it could lead to unexpected errors.

Reply:
Ack.